pdg/src/graph.rs

```diff
@@ -71,7 +81,7 @@ pub struct Node {
     /// operation.  As in `rustc_middle::mir::Location`, an index less than the number of
     /// statements in the block refers to that statement, and an index equal to the number of
     /// statements refers to the terminator.
-    pub index: usize,
+    pub statement_index: usize,
```

Comment:
we can change to `statement_idx` so that elsewhere we don't need to do:
```
statement_index: statement_idx
```

Reply:
Sure, I'll change that.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/469/commits/95d35db08266a55eaf806cf3adec9ae2844fe0eb.